kolibri/plugins/device/assets/src/users/composables/useUsers.js

```diff
@@ -0,0 +1,112 @@
+import UserType from 'kolibri.utils.UserType';
+import { UserKinds } from 'kolibri.coreVue.vuex.constants';
+import { getCurrentInstance, ref, inject, onMounted } from 'kolibri.lib.vueCompositionApi';
+import { TaskResource, FacilityUserResource } from 'kolibri.resources';
+import { TaskStatuses } from 'kolibri.utils.syncTaskUtils';
+import { deviceString } from '../../views/commonDeviceStrings';
+
+const isPooling = ref(false);
+const usersBeingImportedRef = ref([]);
+const users = ref([]);
+const loading = ref(true);
+const showCannotRemoveUser = ref(false);
+
+export default function useUsers() {
```

Comment:
If this is specific to the device plugin, perhaps we should name this something other than `useUsers`. If this is able to be generalized, then perhaps we should consider moving it to `kolibri-common`